DogServiceExample/src/main/java/app/Dog.java

```diff
@@ -1,87 +0,0 @@
-package app;
-
-import java.text.DateFormat;
-import java.text.ParseException;
-import java.text.SimpleDateFormat;
-import java.util.Date;
-
-/**
- * Created by adyachenko on 7/22/2016.!
- */
-public class Dog {
-    private int id;
-    private String name;
-    private Date dateOfBirth;
-    private int height;
-    private int weight;
-
-
-    public int getId() {
-        return id;
-    }
-
-    public void setId(int id) {
-        this.id = id;
-    }
-
-    public String getName() {
-        return name;
-    }
-
-    public void setName(String name) {
-        if (name.length() >= 1 && name.length() <= 100) {
-            this.name = name;
-        } else {
-            System.out.println("Name should has length in range 1-100");
-        }
-
-    }
-
-    public String getDateOfBirth() {
-        DateFormat simpleDateFormat = new SimpleDateFormat("dd.MM.yyyy");
```

Comment:
Store format in the class field -- you duplicate it in getter and setter, could be a mismatch.